### PR TITLE
Add postgrest-sessions-example to Example Apps

### PR DESCRIPTION
**Pull request by monacoremo**
As a first step towards #240

### CODE REVIEW

File: ecosystem.rst [hunk 1]
```diff
@@ -33,6 +33,7 @@ Example Apps
 * `SMRxT/postgrest-demo <https://github.com/SMRxT/postgrest-demo>`_ - multi-tenant logging system
 * `PierreRochard/postgrest-boilerplate <https://github.com/PierreRochard/postgrest-boilerplate>`_ - example auth back-end
 * `marmelab/ng-admin-postgrest <https://github.com/marmelab/ng-admin-postgrest>`_ - automatic database admin panel
+* `monacoremo/postgrest-sessions-example <https://github.com/monacoremo/postgrest-sessions-example>`_ - example for cookie-based sessions
```
Comment (steve-chavez) [inline ecosystem.rst]: How about if you put it first on the list? I'd like to attract more feedback to your sessions example.

Comment (monacoremo) [reply ecosystem.rst]: Sure, would be great to get more input! Done

**Comment by steve-chavez** [comment]
Cool! The change is already live http://postgrest.org/en/v6.0/ecosystem.html#example-apps

**Comment by monacoremo** [comment]
Awesome! Thank you!

**Pull request merged**